### DISCUSSION TITLE
[QUESTION] - Can I use GPT-4o instead of GTP-4 Turbo?

### DISCUSSION THREAD

**Discussion by doruit** [Azure-Samples/graphrag-accelerator#117] [Q&A] [answered]
Hi, i tried to deploy GPT-4 turbo in a number of regions but it seems unavailable.  I'm not sure if it is related but the indexing process keeps failing at 12,5%. 

Can/should i use GPT-4o instead?

**Comment by mohanajuhi166** [comment]
same issue with me

**Comment by rnpramasamyai** [comment]
Same here

**Comment by timothymeyers** [comment, marked as answer]
@doruit @mohanajuhi166 @rnpramasamyai 

You can use GPT-4o, and you can also request a quota increase for GPT 4-turbo.

In Azure OpenAI Studio, under Quotas (left side) there is a column on the far right for "Request Quota" in a provided region. [More on AOAI Quotas here](https://learn.microsoft.com/en-us/azure/ai-services/openai/quotas-limits)

This is not really a bug with the GraphRAG Azure example so much as a resource availability challenge. Suggest we move this conversation to a Discussion.

**Reply by tanusri9494** [reply]
Hey i am trying to access graph rag API in node js. 
I gave the Subscription keys by creating a API management resource group and while i am trying to index it doesn't actually work. 

I thought may be it requires Openai api keys. so i check, and i find there is no way mentioned in the api docs that allows me to Insert Openai keys with an option to select model on my own :)))) 

this is really disappointing @timothymeyers

**Reply by timothymeyers** [reply]
See #124 for continued discussion.

**Comment by rnpramasamyai** [comment]
Please create the Azure OpenAI service in a different region.
